wikiHow/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "author": {
+        "name": "Nekolen",
+        "id": "270125856679002113"
+    },
+    "service": "wikiHow",
+    "description": {
+        "en": "wikiHow - How to do anything!",
+        "ro": "wikiHow - Cum să faci orice!",
+        "pt_BR": "wikiHow - Як зробити що-небудь!",
+        "fr": "wikiHow - Comment faire n'importe quoi!",
+        "uk": "wikiHow - Як зробити що-небудь!",
+        "nl": "wikiHow - Як зробити що-небудь!"
```

Comment:
The translations arent right, if you dont know the translation of it remove the whole line

Reply:
done!